graphene/sys/tests/constant.c

```diff
@@ -21,6 +21,7 @@
                     unsigned long: "%lu", \
                     long long: "%lld", \
                     unsigned long long: "%llu", \
+                    float: "%f", \
```

Comment:
Why adding this line? Just below there the same printf option.

Reply:
GCC (v 10.2.1) emits an error:

```
error: ‘_Generic’ selector of type ‘float’ is not compatible with any association
```

Reply:
Corresponding change in `gir` is [here](https://github.com/gtk-rs/gir/pull/1142).